setTRIMDAC.py

```diff
@@ -506,10 +507,11 @@ def txtTitle(str):
 
             regName = "vfat2_" + str(port) + "_channel" + str(channel + 1)
             trimDAC = (g.readline()).rstrip('\n')
-            print trimDAC
-#            regValue = int(trimDAC) #correct version?
-            regValue = (1 << 6) + int(trimDAC) #old version?
-            #print regValue
+            #print trimDAC
+            regValue = int(trimDAC) #correct version?
+            #regValue = (1 << 6) + int(trimDAC) #old version?
+	    #regValue = (1 << 6) - int(trimDAC)  #BLD version?
+            print regValue
             glib.set(regName, regValue)
 #We should make the S-curve optional, generally not a necessary check 
 #            glib.set('scan_reset', 1)
```

Comment:
I think this was old unmerged/uncommitted changes from when we were doing the slice test QC attempting to troubleshoot.

This script is superseded by confChamber.py with the --chConfig option.

Suggest to remove setTRIMDAC.py